grouper/audit.py

```diff
@@ -1,4 +1,4 @@
-from constants import PERMISSION_AUDITOR
+from .constants import PERMISSION_AUDITOR
```

Comment:
can we make that `from grouper.constants import PERMISSION_AUDITOR` relative imports are confusing. :)

actually, i'll do it and merge it.